app/assets/stylesheets/patternfly_and_overrides.scss

```diff
@@ -123,6 +123,7 @@ a {
   width: 44px;
   height: 34px;
   margin: 0 5px 0 0 !important;
+  max-width: inherit;
```

Comment:
the inherited value is `none` - should we set that explicitly in case one of the parents changes in the future?

Reply:
sure, no problem